client/store/cart.js

```diff
@@ -72,7 +78,7 @@ export const getCart = () => async dispatch => {
 
 export const getUpdateCart = fruit => async dispatch => {
   try {
-    const {data} = await axios.put(`/api/cart/${fruit.fruitId}`, {
+    const {data} = await axios.post(`/api/cart/${fruit.fruitId}`, {
```

Comment:
Not sure if I interpreted the definitions correctly but it seems like we should use put.

PUT - If the Request-URI refers to an already existing resource – an update operation will happen, otherwise create operation should happen if Request-URI is a valid resource URI 

POST - The POST method is used to request that the origin server accept the entity enclosed in the request as a new subordinate of the resource identified by the Request-URI in the Request-Line. 

POST is NOT idempotent. So if you retry the request N times, you will end up having N resources with N different URIs created on server.

https://restfulapi.net/rest-put-vs-post/

Reply:
I decided to change it to a post because adding an item from the product page is a request that the server accept a new fruit. 

But if you insist on changing it back, it should still be fine. the api routes for item update and quantity update is different.